src/library.js

```diff
@@ -23,12 +23,7 @@
 // (using makeStaticAlloc)
 
 LibraryManager.library = {
-  // keep this low in memory, because we flatten arrays with them in them
-#if USE_PTHREADS
-  _impure_ptr: '; if (ENVIRONMENT_IS_PTHREAD) __impure_ptr = PthreadWorkerInit["__impure_ptr"]; else PthreadWorkerInit["__impure_ptr"] = {{{ makeStaticAlloc(4) }}}',
```

Comment:
so `_impure_ptr` is just all in wasm now?

Reply:
Yeah, that was some weird JS detail due to how we handled older versions of musl - this particular code has been dead for quite some time.

Reply:
(There is no more `impure_ptr` in the codebase at all.)